cli/data.go

```diff
@@ -560,7 +560,7 @@ func (c *viamClient) deleteTabularData(orgID string, deleteOlderThanDays int) er
 	return nil
 }
 
-// DataAddToDataset is the corresponding action for 'data dataset add'.
+// DataAddToDataset is the corresponding action for 'data dataset add ids'.
 func DataAddToDataset(c *cli.Context) error {
```

Comment:
rename to DataAddToDatasetByIDs here and below?

Reply:
Yep